change/@ni-nimble-components-ac937d1b-bb11-4937-90be-1d29c446ca0b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Let's add a note to the Storybook docs for the number column to mention that it respects the language token. I think Mert did something similar in the date column docs (possibly only visible in the story source right now) which you could copy from.